src/server/config/cmds.go

```diff
@@ -374,13 +374,22 @@ func Cmds() []*cobra.Command {
 				return err
 			}
 
+			activeEnterpriseContext, _, err := cfg.ActiveEnterpriseContext(true)
+			if err != nil {
+				return err
```

Comment:
UX question: if `errorOnNoActive` is set to `true`, does returning an error here make it impossible to list your contexts if you have no enterprise context set? (does that apply to e.g. open-source users?)

Reply:
I was also curious about this. `ActiveEnterpriseContext`[ is implemented](https://github.com/pachyderm/pachyderm/blob/master/src/internal/config/config.go#L88) so that if no active-enterprise-context is listed, it will return the active-context. 

The `errorOnNoActive` argument actually gets used when checking for the active-context. So when `errorOnNoActive` is set to `true`, an error will only be returned if neither an `active-enterprise-context` nor an `active-context` are set

Reply:
I see. I mean, would it make sense to still print the overall list of contexts with no `E`, even in the case where there's no active context selected? My worry is that right now, if your active context becomes unset, there would be no way to list the contexts for purposes of selecting a new one